cmd/mod.go

```diff
@@ -263,10 +262,11 @@ func newInstallOpts(cmd *cobra.Command, args ...string) *modinstaller.InstallOpt
 // Modifies(trims) the URL if contains http ot https in arguments
 
 func trimGitUrls(opts *modinstaller.InstallOpts) {
-	re := regexp.MustCompile(`^(https?://)`)
-	for i, url := range opts.ModArgs {
-		if re.MatchString(url) {
-			opts.ModArgs[i] = re.ReplaceAllString(url, "")
-		}
-	}
+    for i, url := range opts.ModArgs {
+        if strings.HasPrefix(url, "http://") {
```

Comment:
`strings.TrimPrefix` already does a `HasPrefix` under-the-hood. This check is not required.

```
strings.TrimPrefix(url, "http://")
strings.TrimPrefix(url, "https://")
```

Reply:
Thank you, I didn't knew about that